3/models/autoencoder.py

```diff
@@ -0,0 +1,107 @@
+import torch.nn as nn
+import torch.nn.functional as F
+import torch
+import os
+import numpy as np
+from tensorboardX import SummaryWriter
+
+
+
+class AE(nn.Module):
```

Comment:
May not be the best architecture in general but for the sake of homework is fine